src/components/ItemList/index.jsx

```diff
@@ -1,8 +1,15 @@
 import React, { useState, useEffect } from 'react';
 import { updateItemDate } from '../Utils/firestore';
-import { isWithin24hours } from 'components/Utils/helpers';
-
-function ItemList({ itemName, docId, lastPurchasedDate }) {
+import { isWithin24hours, latestInterval } from 'components/Utils/helpers';
+const { default: calculateEstimate } = require('lib/estimates');
```

Comment:
Yeah. I don't think this follows best practices. 
cost calculateEstimate from 'lib/estimates' 
would be better. 

Just a bit on how imports work, when you see the first line:
import React, { useState, useEffect } from 'react';
the React in there is the 'default' import, you can actually name the default import whatever you want and it would be there, look at how it doesn't have any curly brackets around it. 
Now useState, and useEffect are named "exports/imports" so they are exported in React under that specific name. 
Read more here: https://medium.com/@etherealm/named-export-vs-default-export-in-es6-affb483a0910

Reply:
Hello, the import is done by default from visual Studio Code, I just call the function and import it. Anyway I fixed it, @Indira this way you suggested gives error, change it like this `import calculateEstimate from 'lib / estimates'`.
Good information regarding imports in ES6. Thank you so much 👍